templates/home/dashboard.html

```diff
@@ -417,4 +428,46 @@ <h4 class="text-center mt-3">Volunteers Present</h4>
         }
     }
 </script>
+
+<script src="https://cdn.jsdelivr.net/npm/js-cookie@rc/dist/js.cookie.min.js"></script>
+<script>
+    function hw_done(id, checkbox_id) {
+        var checked = false;
+        $('#' + checkbox_id).change(function(){
+            if($(this).prop("checked") == true){
+               checked = true;
+            } else {
+                checked = false;
+            }
```

Comment:
Why have you created a separate variable for `checked`?  You can just write `'is_homework_done': $(this).prop("checked"),` below and it will work the same.